source/wp-content/themes/wporg-news-2021/sass/components/_local-header.scss

```diff
@@ -133,102 +130,116 @@
 						transform: rotate(180deg);
 					}
 
+					// Display the scrim when toggled.
 					&::before {
-						content: "";
-						display: block;
-						position: fixed;
-						top: 0;
-						left: 0;
-						width: 100%;
-						height: 100vh;
-						background-color: rgba(0, 0, 0, 0.5);
-						z-index: -1;
-						animation-duration: 400ms;
-						animation-name: overlay-fade-in;
+						opacity: 1;
```

Comment:
I think display would work better here - with `opacity` it's always on screen, so it's overlaying the whole content even when the menu is closed.

Reply:
We're able to transition the value of `opacity`, but we can't do that if also has `display: none;`. Are there implication to having it there (but transparent) all the time?

Reply:
Yeah, clicking anywhere on the page opens the menu :)

Reply:
lol, really? How did I miss that... I'll look into it.

Reply:
My bad. Fixed now. `pointer-events` only work when the menu is open now.

Reply:
👍🏻  yep, working for me now